applications/nrf5340_audio/tools/buildprog/buildprog.py

```diff
@@ -35,7 +35,7 @@
 BUILDPROG_FOLDER = Path(__file__).resolve().parent
 NRF5340_AUDIO_FOLDER = (BUILDPROG_FOLDER / "../..").resolve()
 NRF_FOLDER = (BUILDPROG_FOLDER / "../../../..").resolve()
-USER_CONFIG = BUILDPROG_FOLDER / "nrf5340_audio_dk_devices.json"
+USER_CONFIG = BUILDPROG_FOLDER / "nrf5340_audio_dk_devices.json" if os.getenv("AUDIO_KIT_IDS") is None else Path(os.getenv("AUDIO_KIT_IDS"))
```

Comment:
Suggest this is made more readable by:
```suggestion
 if os.getenv("AUDIO_KIT_IDS") is None:
    USER_CONFIG = BUILDPROG_FOLDER / "nrf5340_audio_dk_devices.json" 
 else: 
     USER_CONFIG = Path(os.getenv("AUDIO_KIT_IDS"))
```

Reply:
Also, mind the naming. It is a tad strange that "AUDIO_KIT_IDS are set to "USER_CONFIG".
I think your name is better.  Suggest that both AUDIO_KIT_IDS and USER_CONFIG is renamed to something like AUDIO_KIT_SNRS or AUDIO_KIT_SERIAL_NUMBERS_JSON